### PR TITLE
added more tracking services

### PR DESCRIPTION
**Pull request by mzillich**

### CODE REVIEW

**Comment by mzillich** [comment]
sorry, in a hurry, I will just merge.

File: squirrel_planning_knowledge_msgs/srv/AddObjectService.srv [hunk 1]
```diff
@@ -1,10 +1,4 @@
 # add object to the scene database
-string id
```
Comment (NadiaHG) [inline squirrel_planning_knowledge_msgs/srv/AddObjectService.srv:2]: please readd these lines, squirrel_interface_perception needs them

File: squirrel_planning_knowledge_msgs/srv/UpdateObjectService.srv [hunk 2]
```diff
@@ -1,9 +1,5 @@
 # update an object in the scene database
 squirrel_object_perception_msgs/SceneObject object
-string id
```
Comment (NadiaHG) [inline squirrel_planning_knowledge_msgs/srv/UpdateObjectService.srv:3]: please readd these lines, squirrel_interface_perception needs them

**Comment by ipa-nhg** [comment]
sorry @mzillich  but I have to revert the merge!

**Comment by mzillich** [comment]
@ipa-nhg my pending PR in planning  #49 solves this, so if we merge both, things are fine

**Comment by ipa-nhg** [comment]
No, at least hier:  https://github.com/mzillich/squirrel_planning/blob/indigo_dev/squirrel_interface_perception/src/RPPerceptionAction.cpp#L154 (mzillich:indigo_dev), `AddObjectService` needs still id, category, pose ...

**Comment by ipa-nhg** [comment]
Ups, yes you are right my bad, you use a SceneObject object with category, id...

Sorry!

**Pull request merged**